examples/parsons_civis_twilio_example.py

```diff
@@ -0,0 +1,78 @@
+# This script is an example of using the parsons library to import data from
+# Twilio using the parsons Twilio connector, and then upload the resulting
+# table to Platform using the parsons Civis connector.
+# Below are instructions to be able to customize and run this file in Platform.
+
+# Step 1: Copy this file to a github repository that is connected to Platform.
+#         For docs on how to connect Platform to Github, follow the steps in
+#         the first two paragraphs under "Connecting Platform to Github/Bitbucket":
+#         https://civis.zendesk.com/hc/en-us/articles/115003734992-Version-Control
+
+# Step 2: Set CIVIS_DATABASE and CIVIS_TABLE global variables below
+#         List of databases: platform.civisanalytics.com/spa/remote_hosts
+#         CIVIS_TABLE should be in "schema.table" form
+
+CIVIS_DATABASE = 99
+CIVIS_TABLE = "twilio_test.account_usage"
+
+# Step 3: Create a platform custom credential with the following fields:
+#         Name: Twilio Account
+#         Credential Type: Custom
+#         Username: Twilio Account SID
```

Comment:
put angle brackets around these or something to indicate that they're not literals?